tests/integration/api/controllers/places.spec.js

```diff
@@ -132,16 +166,16 @@ describe('Places API', () => {
     });
 
     it('should create place with contact uuid', () => {
+      const contactId = 'fixture:user:online';
       onlineRequestOptions.body = {
         name: 'DS',
         type: 'district_hospital',
-        contact: 'fixture:user:online'
+        contact: contactId
       };
       return utils.request(onlineRequestOptions)
         .then(result => {
           chai.expect(result.id).to.not.be.undefined;
-          chai.expect(result.contact.id).to.not.be.undefined;
-          return utils.getDocs([result.id, result.contact.id]);
+          return utils.getDocs([result.id, contactId]);
```

Comment:
Is this changing because now we're _not_ returning this contact id as part of the response? 
I think we should add an assertion for that, so we make the API response very clear in our tests.

Reply:
Yes, the contact id is now only returned if we create a contact. I did not think this would be a breaking change but will revert so the previous behavior doesn't change